lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -128,38 +129,34 @@ def __init__(self, daemon):
         self.daemon = daemon
 
     def __call__(self):
-        d = self._get_lbrynet_version()
+        d = threads.deferToThread(self._get_lbrynet_version)
+        d.addErrback(self._trap_and_log_error, 'lbrynet')
         d.addCallback(lambda _: self._get_lbryum_version())
```

Comment:
I don't think once the original function that was called in deferToThreads, control goes back to the reactor this. that means that get_lbryum_version happens synchronosly and will block the reactor thread until a response has come back.